src/plugin.rs

```diff
@@ -50,18 +46,10 @@ impl Plugin for SvgPlugin {
     fn build(&self, app: &mut App) {
         app.add_asset::<Svg>()
             .init_asset_loader::<SvgAssetLoader>()
-            .configure_set(
-                Stage::SVG.after(CoreSet::PostUpdate)
-            )
-            .add_system(
-                svg_mesh_linker.in_base_set(Stage::SVG)
-            )
-            .add_system(
-                origin::add_origin_state.in_base_set(Stage::SVG)
-            )
-            .add_system(
-                origin::apply_origin.in_base_set(CoreSet::PostUpdate)
-            )
+            .configure_set(Stage::SVG.after(CoreSet::PostUpdate))
```

Comment:
I ran cargo format which seems to have changed some of this stuff. :)

Reply:
I've noticed that and also that i missed doing it when updating to 0.10 😅